tdms/include/simulation_manager/simulation_manager.h

```diff
@@ -0,0 +1,103 @@
+/**
+ * @file simulation_manager.h
+ * @brief Class that runs the physics of the TDMS executable, and produces an output.
```

Comment:
Worth writing that the implementation is split: `simulation_manager.cpp` *and* in `execute_simulaion.cpp`?